src/main/java/seedu/ccacommander/logic/commands/EditEventCommand.java

```diff
@@ -81,6 +82,21 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_EVENT);
         }
 
+
+        Name prevName = eventToEdit.getName();
+        Name newName = editedEvent.getName();
+        // If event's name is edited, the corresponding enrolment objects are edited also
+        if (!prevName.equals(newName)) {
+            List<Enrolment> enrolmentList = model.getFilteredEnrolmentList();
```

Comment:
if we use getFilteredEnrolmentList() here will we only edit the currently filteredEnrolmentList?